src/disaster_vehicle.cpp

```diff
@@ -578,17 +579,27 @@ static bool DisasterTick_Big_Ufo(DisasterVehicle *v)
 		}
 		v->current_order.SetDestination(1);
 
-		TileIndex tile_org = RandomTile();
-		TileIndex tile = tile_org;
-		do {
-			if (IsPlainRailTile(tile) &&
-					Company::IsHumanID(GetTileOwner(tile))) {
```

Comment:
AIs are not really able to manage UFO effects, that's why only human owned tile were targeted.
I think the new method should also filter out AI owned vehicles.